debian/rules

```diff
@@ -1,7 +1,6 @@
 #! /usr/bin/make -f
 
 #export DH_VERBOSE = 1
-export PYBUILD_NAME = pyndn
```

Comment:
Are you sure it’s safe to remove this? All the [documentation](https://wiki.debian.org/Python/Pybuild#line-33) seems to say it’s required; and looking at `/usr/bin/pybuild` the code seems to set the package name to `PYBUILD_NAME_not_set` if the variable is not set.